Tools-Override/tests.targets

```diff
@@ -214,6 +214,45 @@
       <PostExecutionTestCommandLines Include="copy /y testResults.xml %EXECUTION_DIR%\" />
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
+    </PropertyGroup>
+  
+    <!-- The results path is different on the helix machine than locally -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true' and '$(ArchiveTests)' == 'true'">
+      <_Results_Path>%USERPROFILE%\Documents\TestResults\</_Results_Path>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
```

Comment:
Might be worth merging these PropertyGroups if we can avoid the folk based on ArchiveTests